modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/task/Task.java

```diff
@@ -198,6 +202,14 @@ public boolean isEnabled() {
         return enabled;
     }
 
+    public void setRetryTaskOnFailure(boolean retryTaskOnFailure) {
+        this.retryTaskOnFailure = retryTaskOnFailure;
+    }
+
+    public boolean getRetryTaskOnFailure() {
```

Comment:
getter convention for primitive boolean is 'is...' so it should be 'isRetryTaskOnFailure'